walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/index.html

```diff
@@ -1,27 +1,88 @@
 <!DOCTYPE html>
-<html>
+<html lang="en" dir="ltr">
   <head>
-    <meta charset="UTF-8">
-    <title>Stanley 🐈</title>
+    <meta charset="utf-8">
     <link rel="stylesheet" href="style.css">
+    <title>Raymond's Portfolio</title>
   </head>
   <body>
-    <div id="content">
-      <h1>Stanley 🐈</h1>
-      <p>This is Stanley.</p>
-      <img src="images/stanley-1.jpg"/>
+    <div class="header-container">
+        <a class="header-item" href="#home">Raymond C. Alvarez Velez</a>
+        <a class="header-item" href="#about">About</a>
+        <a class="header-item" href="#work">My Work</a>
+        <a class="header-item" href="#contact">Contact</a>
+    </div>
+    <div class="content-container">
+        <div class="about-container">
+            <div class="image">
+              <img src="images/stanley-11.jpg" alt="">
+            </div>
+            <div class="description">
+              <p> Hi! My name is Raymond</p>
+              <p> I'm an intern at Google within the Travel Packages Team based in Cambridge, Massachusetts</p>
+              <p> My favorite things to do are exploring nature, target practice, and programming</p>
+              <p> In this portfolio you will find the work I have done!</p>
+              <p> The main CSS concepts I use here are flexbox and media queries</p>
+            </div>
+        </div>
+        <div class="work-container">
+
```

Comment:
nit: please remove unnecessary newlines. It's harming readability more than helping.

As a general rule of thumb, don't add a newline if the next line is not of the same indentation level, and you can optionally add one if it is of the same indentation level.

Good:
```html
<div>
  <div></div>
  <div></div>
  
  <div class="but-its-different"></div>
</div>
```

Bad:
```html
<div>

  <div>

   </div>

</div>
```

Reply:
done